share/wake/lib/system/job.wake

```diff
@@ -268,6 +272,8 @@ export def mkJobCacheRunner ((Runner name score baseDoIt): Runner): Runner =
 
         require False = cacheHit
         else
+          def _ = write ".cache-hit/read.{prefix}.json" "//{label}\n{jobCacheJsonIn}"
+          def _ = write ".cache-hit/out.{prefix}.json" "//{label}\n{cacheResult}"
```

Comment:
Is this for debugged or does it serve another purpose?

Reply:
Its for debugging. I should probably wrap it in an environment variable.